### PR TITLE
VM scripts

### PR DESCRIPTION
**Pull request by Shillaker**
Some changes around how we do VMs:

- Add scripts for creating stand-alone VMs for running experiments or development.
- Remove old `az-vm` scripts that are no longer used.

### CODE REVIEW

File: tasks/util/env.py [hunk 1]
```diff
@@ -11,16 +11,29 @@
...
-# AKS_CLUSTER_NODE_COUNT = 4
-
-# 2 8-core machines: migration experiment w/ 8 parallel functions
-# AZURE_VM_SIZE = "Standard_D8_v5"
-# AKS_CLUSTER_NODE_COUNT = 2
+KUBECTL_BIN = join(PROJ_ROOT, "bin", "kubectl")
 
-# 1 16-core machine: LULESH experiment
-AZURE_VM_SIZE = "Standard_D16_v5"
+AZURE_PUB_SSH_KEY = "~/.ssh/id_rsa.pub"
```
Comment (csegarragonz) [inline tasks/util/env.py:16]: I am confused, how will this work now? Will it use the machine's key's for accessing the VM?

File: tasks/cluster.py [hunk 2]
```diff
@@ -57,9 +53,10 @@ def provision(ctx):
         [
             "--name {}".format(AKS_CLUSTER_NAME),
             "--node-count {}".format(AKS_CLUSTER_NODE_COUNT),
-            "--node-vm-size {}".format(AZURE_VM_SIZE),
+            "--node-vm-size {}".format(AZURE_K8S_VM_SIZE),
+            "--os-sku Ubuntu",
```
Comment (csegarragonz) [inline tasks/cluster.py:57]: I never worried about setting this parameter, did you find it necessary?

**Review by csegarragonz — APPROVED** [review]

File: tasks/util/env.py (hunk 1 above)
Comment (Shillaker) [reply tasks/util/env.py:16]: I assume it just adds your public key to the authorized keys for the machine. I think it was doing this anyway but wasn't sure, plus I sometimes create new key pairs for stuff like this so wanted to make it configurable.

File: tasks/cluster.py (hunk 2 above)
Comment (Shillaker) [reply tasks/cluster.py:57]: It's not necessary as I think it's the default, but I couldn't work out what OS we were using and wanted to be explicit about Ubuntu in the hope that I could put other SKUs in here. We need to have at least 20.04, preferably with HWE installed, so I want to fiddle with this a bit more in future.

**Pull request merged**